<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblServer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledServer">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblDatabase">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Database</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledDatabase">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblUser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledUser">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledPassword">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbConnect">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>256</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDisconnect">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit from application</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
